nikola/nikola.py

```diff
@@ -1984,6 +1984,20 @@ def create_hierarchy(cat_hierarchy, parent=None):
         # Next, flatten the hierarchy
         self.category_hierarchy = utils.flatten_tree_structure(root_list)
 
+    def sort_posts(self, posts, lang=None):
```

Comment:
This is a name clash with `nikola.utils.sort_posts`. It should be named something distinctive, eg. `sort_posts_chronologically` (its main purpose)

Moreover, the sort should be done in place (that’s what we did before)